app.js

```diff
@@ -0,0 +1,84 @@
+
+   
+
+    function regEx(){
+    function testName(){
+            let name = document.getElementById('username').value;
+            let nameTest = name.search(/[a-zA-Z]/g);
+            if (nameTest == -1 || name == ""){
+                alert ("please enter only letters here");
+            };
+
+    }
+    function testEmail(){
+            let email = document.getElementById('email').value;
+            let emailTest = name.search( /^\w+([\.-]?\w+)*@\w+(\.-]?w+)*(\.\w{2,3})+$/);
+            if (emailTest == -1 || email == ""){
+                alert ("please enter valid email here");
+            };
+
+    }
+    testEmail();
+    testName();
+    insetName();    
+    
+    }
+
+    
+function validate(){
+	
+	let username1 = document.getElementById("username").value;
+	let password1 = document.getElementById("password").value;
+ 
+	let name = ["cris","doe"];
+    let pass = ["123","111"];
+    
+
+	for (let i = 0; i < name.length; i++) {
+		
+		if (username1 == name[i] && password1 == pass[i]) {
+		alert ("Succefully Logged in!");
+		window.location = "dashboard.html";
+		return false;
+	}
+	else{
+		
+		alert ("invalid user name or password");
+		document.getElementById("submit").disable = true;
+	}
+regEx();
+
+}
+}
+
+
+    const data = new Array();
+    //const data = [];
+   

```

Comment:
l don't think this line is not necessary

Reply:
but i am creating an empty array to store the data